src/seedu/addressbook/ui/TextUi.java

```diff
@@ -156,7 +157,20 @@ private static String getIndexedListForViewing(List<String> listItems) {
         }
         return formatted.toString();
     }
-
+    
+    /**
+     * Returns the concated version of the printable strings from each Object 
+     */
+    
+    String getPrintableString(Printable... printables){
```

Comment:
answering your uncertainty why should this method be in `TextUi`. I don't think the LO mentioned to put this method here. It's probably in the Address class so that when you call `Address.getPrintableString()`, it can call the overloaded method `getPrintableString(Printable... printables)` and return the final string.